web/packages/teleport/src/Integrations/Enroll/AwsOidc/AwsOidc.tsx

```diff
@@ -178,23 +104,33 @@ export function AwsOidc() {
       <Validation>
         {({ validator }) => (
           <>
-            <Container mb={5}>
+            <Container mb={5} width={800}>
               <Text bold>Step 1</Text>
               <Box width="600px">
                 <FieldInput
                   autoFocus={true}
-                  value={integrationName}
+                  value={integrationConfig.name}
                   label="Give this AWS integration a name"
                   placeholder="Integration Name"
-                  onChange={e => setIntegrationName(e.target.value)}
+                  onChange={e =>
+                    setIntegrationConfig({
+                      ...integrationConfig,
+                      name: e.target.value,
+                    })
+                  }
                   disabled={!!scriptUrl}
                 />
                 <FieldInput
                   rule={requiredIamRoleName}
-                  value={roleName}
-                  placeholder="IAM Role Name"
-                  label="IAM Role Name"
-                  onChange={e => setRoleName(e.target.value)}
+                  value={integrationConfig.roleName}
+                  placeholder="Integration role name"
+                  label="Give a name for an AWS IAM role this integration will create"
```

Comment:
It was confusing previously if we were asking for a new role name or an existing role name. https://github.com/gravitational/teleport/issues/45387